onebusaway-android/src/main/res/layout/trip_plan_advanced_settings_dialog.xml

```diff
@@ -13,64 +13,88 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:orientation="vertical"
-              android:layout_width="fill_parent"
-              android:layout_height="fill_parent"
-              android:paddingLeft="@dimen/dialog_margin"
-              android:paddingRight="@dimen/dialog_margin"
-              android:paddingTop="@dimen/dialog_margin">
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingStart="@dimen/dialog_margin"
+    android:paddingEnd="@dimen/dialog_margin"
+    android:paddingTop="@dimen/dialog_margin">
 
-    <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="horizontal">
+    <TextView
+        android:id="@+id/travel_by_textview"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/travel_by_label"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/travel_by_label" />
+    <Spinner
+        android:id="@+id/spinner_travel_by"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/travel_by_textview"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <Spinner
-                android:id="@+id/spinner_travel_by"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content" />
+    <TextView
+        android:id="@+id/textView3"
+        android:layout_width="0dip"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/text_margin"
+        android:layout_marginEnd="8dp"
+        android:text="@string/maximum_walk_distance"
+        app:layout_constraintEnd_toStartOf="@+id/layout_maximum_walk_distance"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/travel_by_textview" />
 
-    </LinearLayout>
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
Maintainer Note :
Replaced EditText with TextInputLayout as said by #537

Reply:
Maintainer Note:
No speakable text present

From Accessibility side (Content Description) what should text should be added here?